modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,69 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage"]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+Before you modify the machine set, you must manually generate the `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Installed the OpenShift CLI (`oc`).
+* Configured your Azure portal account.
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
```

Comment:
not necessarily a prerequisite to have oc and a cluster installed to generate an arm bootimage.

Reply:
Ah, that was my mistake when messing with the modules. Thank ya!